packages/mpl-scatter-density.yml

```diff
@@ -0,0 +1,3 @@
+repo: astrofrog/mpl-scatter-density
+section: miscellaneous
```

Comment:
I think this and datashader are plot types ?

Reply:
Yes and no, but sure!

```suggestion
section: plot types
```